metricbeat/module/mongodb/status/status.go

```diff
@@ -17,43 +15,52 @@ TODOs:
 	* add a metricset for "metrics" data
 */
 
-var debugf = logp.MakeDebug("mongodb.status")
```

Comment:
I assume you removed this, because it was not used. We normally keep this in to make it easy to add debug messages.